harness/determined/pytorch/dsat/_run_dsat.py

```diff
@@ -54,14 +70,19 @@ def main(core_context: det.core.Context) -> None:
     args.config_path = os.path.basename(args.config_path)
     args.model_dir = os.path.basename(args.model_dir)
     args.include = [os.path.basename(p) for p in args.include] if args.include is not None else []
-    args.experiment_id = det.get_cluster_info()._trial_info.experiment_id
+    cluster_info = det.get_cluster_info()
+    if cluster_info is None or cluster_info._trial_info is None:
+        return
+    args.experiment_id = cluster_info._trial_info.experiment_id
 
     exp_config = get_custom_dsat_exp_conf_from_args(args)
 
-    search_method_class = _defaults.ALL_SEARCH_METHOD_CLASSES[args.search_method]
+    search_method_class = get_search_method_class(args.search_method)
+    if search_method_class is None:
```

Comment:
Doesn't seem like `search_method_class` can be `None`, given the form of `get_search_method_class`?

Reply:
That's true, I can get rid of this now. It didn't in previous versions of the code